GOVERNANCE.md

```diff
@@ -28,8 +26,14 @@ No more than one-fourth of the CC members may be affiliated with the same employ
 
 The CC members shall consist of active members of Community Projects and the two Individual Membership Directors as defined in Section 10.
 
+The CC may, at its discretion, invite any number of non-voting observers to participate in the public portion of CC discussions and meetings.
```

Comment:
In this context, The Community Committee is an entity chartered by the Node.js Foundation, not a group of people. People run the CC, but they are not the CC, legally speaking.

> How would a member of the CC go about inviting a non-voting observer to participate in the public portion of the CC discussions/meetings? Would they need to create an issue on GitHub requesting or notifying of the invitation, or is it entirely open for any CC member to invite a non-voting observer at their discretion?

Yeah, we _definitely_ do not want to specify that in the charter. If it turns out that what we specify doesn't work, it will take a long time to fix it. We learned the hard way in the TSC that it's better to leave the charter vague, and then back it up with details in other documents that aren't the charter and can be changed easily.